static/images/icons/FontAwesome/medal-solid.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
```

Comment:
Flawless! Good job using the bare minimum (for attributes). This is exactly how I would've defined the asset.